src/lib/menu/ClientMenuActionHandler.ts

```diff
@@ -0,0 +1,67 @@
+import { ipcRenderer } from 'electron'
+import _ from 'lodash'
+import AppEvent from '../../common/AppEvent'
+
+function send(name: string, arg?: any) {
+  ipcRenderer.send(AppEvent.menuClick, name, arg)
+}
+
+export interface IClientMenuActionHandler {
```

Comment:
Can you combine this with an interface shared by this and the NativeMenuActionHandler? The interface can go in common/

Reply:
Do you mean something like this?

```
type IMenuActionHandler = IClientMenuActionHandler | INativeMenuActionHandler
```

But what is the diference between `IClientMenuActionHandler` and `INativeMenuActionHandler`?

Reply:
Ah, yea I wasn't clear here. There's no difference! They should use the same interface

Both NativeMenuActionHandler and ClientMenuActionHandler have the same interface. So there should be 3 files:

```
common/.../IMenuActionHandler
app/.../ClientMenuActionHandler (implements IMenuActionHandler)
background/.../NativeMenuActionHandler (implements IMenuActionHandler)
```